molgenis/molgenis-emx2
# feat: add columnType=HEADING to enable some layout in your forms

Motivation: users often want to partition large forms into smaller subsets using headers. 

How to test:
* in schema editor you can choose columnType == HEADER
* when viewing forms you should see these headers
* in table / filter view you should not see these headers

Notes:
* also contains a few fixes for ONTOLOGY and ONTOLOGY_TERM
* also fixes bug in TypeUtils.java (or is this a feature, that getBool('yes') == true and getBool('no') == false
* renamed 'constant' to 'heading' which I think is easier to understand. Considered 'seperator' as alternative.

N.B. todo is also show these headers in the 'RECORD' view inside the explorer. The file apps/styleguide/src/tables/RowPage.vue is a draft therefore but still needs to be integrated
